src/parse.c

```diff
@@ -412,13 +412,16 @@ static size_t Parse_TypeSize(const parse_type_t type) {
 		return 4;
 	case PARSE_DOUBLE:
 		return 8;
+	default:
+		signal(SIGSEGV, NULL);
```

Comment:
I guess this will trigger a Com_Error and Sys_Backtrace without introducing a circular dependency there.. I dig it.